### ISSUE TITLE
Issue with loading the mod

### ISSUE THREAD

**Issue by warlord4107** [long-war-2/lwotc#174]
I attempted to test the game removed all the rested mods but when I start the game up it crashes before getting the main menu, it gets to the mod menu so I figured it was one of my other mods so I disabled everything else and it still crashes I have a AMD Ryzen 5 3.5ghz 32 gigs of ram and a 4gig  Radeon video card the game works fine with my other mods but like I said crashes with the new alpha based here is a copy of the error log, would like to become a regular tester for this mod since I've been dying to see it, since LW2

**Comment by warlord4107** [comment]
[Launch-backup-2019.05.20-19.22.26.log](https://github.com/long-war-2/lwotc/files/3204165/Launch-backup-2019.05.20-19.22.26.log)

**Comment by jmartinez989** [comment]
When you say you tried to test the game, do you mean you downloaded a pre built copy of LW2 or did you try to pull the code from git hib and try building it? Reason I ask is because if got a pre built copy of lwotc without the community highlander then it isn't going to work.

**Comment by pledbrook** [comment]
That crash looks like it's happening in the template modification code. That suggests a conflict with another mod you have installed, although I can't say that for sure. You do have quite a lot that seem to be loading up.

**Comment by warlord4107** [comment]
No I was trying to run the alpha version, as for a conflict maybe when I 1st ran it that might be what the crash log is showing but I also ran it again with all my mods disabled except for the highlander mod it came with and the alpha and it still crashed on me.

**Comment by jmartinez989** [comment]
Try this as sometimes the game will try to load mods even if you disable them. Go into your xcom folder under your "My Documents folder" (something like C:\Users\<your user name>\Documents\my games\XCOM2 War of the Chosen\XComGame). In that folder there is a config folder. Delete that folder and then try loading again with no mods. The game will build a fresh copy of that config folder again when you start so don't worry about deleting it, you  may just have to re apply some of your graphics and game settings.

**Comment by pledbrook** [comment]
The _profile.bin_ file in the _SaveData_ directory can also be problematic. You should move that out of the _SaveData_ directory to a safe location and then try. In fact, robojumper has suggested removing everything in the _my games\XCOM2 War of the Chosen_ folder (after creating a backup copy somewhere else on the filesystem) as a "nuclear" option that might also work.

If you're not keen on trying that, post the log output for a launch of the game that you're sure doesn't have other mods enabled.

**Comment by warlord4107** [comment]
Ok, I have over 400+ mods installed so I'm gonna just disable the mods from the mod window and keep just the highlander & alpha selected if it crashes again I will send a new log file up. Btw do they know how long it will be before the alpha is ready to be on steam as a normal download?? also is there any other mods I might need to disable that could effect the game not listed in the install directions?

**Comment by warlord4107** [comment]
Question, I think its working now I cleaned out the config folder, but when I start a new game everything looks the same unlike with LW2 where the mission screen was different and all how can I tell LW3 is actually running and how do you modify your squad size since robojumpers mode is built in. I mean will it say new game or Long War 3? Please advise I'm exicted to see it but would like to know how I can tell its actually Lw3 and not just a normal game.

**Comment by jmartinez989** [comment]
LW3? The mod is lwotc (Long War of the Chosen) :P. In all seriousness the way you can tell is at the main screen where you launch a new game. If the button on the bottom left says "New Game" instead of Long War 2 then long war 2 is not running.

**Comment by warlord4107** [comment]
Damn it doesn't say it I download the LongWar2_WOTC-1.6-alpha-2 and I have it listed in the list of mods that are running and removed all the mods that where mentioned in the txt file I don't understand why its not showing up I did just download a file called I-WOTC-Content.zip is that need to make it work correctly??

**Comment by jmartinez989** [comment]
If you close the game and restart it does it still not say it?

**Comment by warlord4107** [comment]
I haven't tried that but I will give me a few mins should I wipe out the config folder again or leave it before restarting the game?

**Comment by jmartinez989** [comment]
you should only need to have deleted it once. Try a restart. that should work.

**Comment by warlord4107** [comment]
Also is that content folder required because I didn't realize it was there and I see robojumpers squad select in there should I copy it into the longwarwotc folder?

**Comment by warlord4107** [comment]
If your wondering what I'm referring to its this which I download from github:

Building and running the mod

Building and running this mod requires several steps:

    The media assets (video, graphics and sound) are kept as a zip on Dropbox. Download that file and unpack it into this project's LongWarOfTheChosen/Content directory.

**Comment by warlord4107** [comment]
Nevermind I see the files are there already but after I exited the game and restarted it, it crashed I checked all the settings but yet it didn't load, I hope your not going to tell me I have to unsubscribe from my mods I have over 400 installed here is all log files related to the crash please advise I do wish to keep working on getting it running so if you see something I should change let me know please!!
[ctpHttp.log](https://github.com/long-war-2/lwotc/files/3213630/ctpHttp.log)
[FileIO.log](https://github.com/long-war-2/lwotc/files/3213631/FileIO.log)
[Launch.log](https://github.com/long-war-2/lwotc/files/3213632/Launch.log)

**Comment by pledbrook** [comment]
It's still crashing in the same place, and I'm afraid it's still highly likely to be a mod conflict. I don't know why, but at least some of your other mods are being loaded.

Just as an FYI, the crash is happening when LWOTC attempts to modify explosive damage (it applies damage falloff).

The only other suggestion I have is to try the nuclear option I mentioned earlier. Are you using the Alternative Mod Launcher to start the game?

**Comment by warlord4107** [comment]
Do you know which mod might be causing the issue? I removed the jedi robes mod, coilguns mod, the mod that makes the sky ranger look like the drop ship from warhammer and any extra maps.I did this because I looked through the logs and saw errors next to a few of them and I know that maps sometimes cause errors as well. I haven't tried loading it yet, I will attempt the nuclear option, as for using a different launcher no just using the one the game comes with. Should I try the other launcher? and I'll move the profile to my 2nd HD along with a backup of the folders robojumper recommended but it be a lot easier if I knew what was causing the crash in the 1st place even after I wipe out the config folder and only have the highlander mod and lwotc selected it still crashes on me or it loads into the game as if all the mods are there I know it's doing that because it loads the mod config menu under options. As for damage fall off I don't even get that far the game crashes before it even gets to the screens that show unreal or anything else it never makes it, in fact it crashes 2 mins or so into the loading process.

**Comment by pledbrook** [comment]
robojumper identified the likely source of the crash and suggested a workaround. I'll try to implement that, which would then go into alpha 3. You will hopefully then see in the log which mod is causing the conflict (and it shouldn't crash).

**Comment by warlord4107** [comment]
Should I still attempt the nuclear option? or just wait for Alpha 3? also how will I know which mod is causing the issue?? Still kindy new at coding and what to look for but I'm a quick study. Do you have any suggestions? Also is robojumpers work around because of something he saw in my log file?

**Comment by pledbrook** [comment]
I would still try deleting the whole "my games\\XCOM2 War of the Chosen" directory after creating a backup of it. There are no guarantees as to when alpha 3 will be out, although I hope soon™.

I'll post the log message to look for on this issue when it's added. As for the workaround, I don't know how he picked up on it.

**Comment by warlord4107** [comment]
Ok so delete all sub folders inside of the my games\xcom2 war of the chosen directory? if so I'll do that next and see if it fixes the issue if not I will let you know and I'll also continue looking through the log files for where the errors, one question will the game rebuild the folders after they're deleted? I'm sure the answer is yes but I'd rather be safe then sorry :) btw your making me wait because I'm not going to play the game until I see that Long War button :)

**Comment by pledbrook** [comment]
Yes, delete those folders. The game will restore the folders, but it won't restore the saved games, logs, photobooth, etc, which is why you need to back them up.

**Comment by warlord4107** [comment]
I don't care about the photobooth images and saved games I already erased them earlier but I will make the backup anyway as a safety net also should I move the profile file completely from the folder and not have it in there?

**Comment by pledbrook** [comment]
Remove everything, but back up _profile.bin_ because it contains your profile information, which you may want. Things like settings, whether you've gone through the tutorials, etc.

**Comment by warlord4107** [comment]
OK I deleted everything and of course made a backup but unfortunately it didn't solve the problem :( I'm trying the file repair in steam to see if it helps I don't understand what could be causing it, do you know of any other types of mods I might need to avoid to see if I have one that might be causing the issue? 65% of my mods are mostly for the way soldiers look and voice packs don't mess with a lot of the major stuff, and I try to stay with things that are compatible, could things that work with sparks be a cause or maybe even a pocket fix or something like that?

**Comment by warlord4107** [comment]
Never mind I got it to work I see the Long War button instead of the new game also it seems like you need to run the file repair in steam before running the game, I might have had a corrupted file system in the game. I will start testing at once, it also looked extremely fast which was usually for my system normally takes 4 mins from the time it gets off the mod screen this time it only took like maybe a minute I'll keep you advised if any issue start popping up thanks for all your help and damn please keep it rolling I can't wait to see the finished product!!!!

**Comment by warlord4107** [comment]
And nevermind again after I ran the game I realized that the mods where not installed, for some reason so I restarted the game making sure the mods where all selected and it crashed again :( do you think if I sent you the full list of the mods I'm using that you could help me track down the one that's causing the game not to run correctly? I've burnt out my eye balls trying to find the mod that might be causing the problems in the log file I've removed mods like a better start and better arsenal and it still keeps dropping me, like I mentioned really want to play LWOTC but need I need to fix the issue 1st.

**Comment by pledbrook** [comment]
So wait, are you trying to get LWOTC running with as many of your mods enabled as possible? Does it at least work without those other mods enabled?

If you want to track down the mod that results in the crash, then you should wait for alpha 3. Unless you can tell whether one of your mods is likely to be creating or overriding explosive damage effects.

**Comment by warlord4107** [comment]
I'd like LWOTC to work with all of the mods I have but I'll look into the explosive damage effects to see which one might be what I'm looking for, I'm not sure I have one that does that but just in case I'll  check all my mods it sucks because I'm a huge G.I.Joe, Marvel, DC, buff and have tons of characters based on them in my game. But let me look and see will keep you updated.

**Comment by pledbrook** [comment]
I would look for anything that adds a new weapon (not just new cosmetics for an existing weapon) or new abilities.

**Comment by warlord4107** [comment]
Ok so any mod that adds new weapons or new abiilities, ok I'll look into my list for that as well I hope by the time its ready for steam I can put back most of my stuff or do you think that won't be wise?

**Comment by warlord4107** [comment]
Ok I removed most of the mods that you where referring to I added them to a list can you look them over and let me know which I can put back and which could've been causing the issue? I'd really appreciate the help and in return will keep looking for bugs and help where I can. Here is the list:
[removed mods xcom 2.txt](https://github.com/long-war-2/lwotc/files/3218293/removed.mods.xcom.2.txt)

**Comment by warlord4107** [comment]
Hi pledbrook, good news I got it running again and I'm slow re-adding the mods in that list one at a time until I find the one that breaks the game. So far I'm up to number 10 :( still some 100+ to go but all the mods that where there before I started seem to work fine with LW so once I come across the one that breaks it I'll let you know which one so maybe when you do the upgrade to alpha 3 you can find a work around. Like I said I'm very willing to help you test the alpha since I use so many different mods, of course I really don't want to un-install and then re-add them again :)

**Comment by pledbrook** [comment]
@warlord4107 Let me know if alpha 3 fixes the crash.

**Comment by warlord4107** [comment]
Well good news after 2 days and testing over 119 mods here are the 4 I found that cause the game to crash:
Skirmisher Heretic Hero Class
Mitzruti Perk Pack
Ericpar's HealBomb
MEC Troopers

Is alpha 3 ready for downloading? if so please provide link and I will download it hopefully the replacement will not require me to remove more mods.

**Comment by pledbrook** [comment]
Respect, and glad you found out the conflicting mods. You can download alpha 3 by following [these instructions](https://github.com/long-war-2/lwotc/wiki/Play-testing-Long-War-2-%5BWOTC%5D).

**Comment by warlord4107** [comment]
Just so you know it crashed before I got to start the 2nd mission this is using alpha 2, I will restart the game using alpha 3 and see if I have any troubles. If you wish I can send you the crash log file.

**Comment by warlord4107** [comment]
Pledbrook, still crashes when you attempt to go on your 1st raid not the one in the beginning but the next one and this happen with both alpha 2 & 3 here is the log files:
[ctpHttp.log](https://github.com/long-war-2/lwotc/files/3220969/ctpHttp.log)
[FileIO.log](https://github.com/long-war-2/lwotc/files/3220970/FileIO.log)
[Launch.log](https://github.com/long-war-2/lwotc/files/3220971/Launch.log)

**Comment by pledbrook** [comment]
The log file isn't particularly helpful I'm afraid. A save file would be better, so I can try to reproduce the crash. Also, at what point exactly does it crash? As soon as you click on Launch Mission? During the dropship briefing screen? Later?

**Comment by warlord4107** [comment]
Just you click on launch mission, I'd send you the saved game but it is saved prior to the crash, so I'm not sure what help it will be. But if you still want it I will send it to you. Actually yes its after the drop ship leaves and you can then click on start mission or whatever its called that's when it dumps me out.

**Comment by warlord4107** [comment]
@pledbrook here is the saved game file, let me know if you have any luck figuring out whats wrong with it.

[save_New Battle.zip](https://github.com/long-war-2/lwotc/files/3224077/save_New.Battle.zip)

**Comment by Hound714** [comment]
Little late in commenting here, I think the issue is with resistance havens/personnel, the crash happens on when I start the jailbreak mission (after the mission has loaded and it gives me the "press any key to continue")  or try to manage a resistance haven.  It's definitely a mod conflict.  @warlord4107 I discovered a ton of the mods I had were already integrated into LWOTC. While I don't have 400+ mods, once I essentially disabled EVERY mod outside of highlander and LWTOC, the jailbreak and haven management worked. Hate to say it but you might have to start a new game and then slowly enable mods. Sort of good news is you can enable multiple mods at a time if they do the same thing, like voice mods.

**Comment by Hound714** [comment]
Well, I was wrong, every mod I tried from, voice mods, to cosmetics, to some of the QOL mods, crash the jailbreak mission/haven management.  I purged everything on each new game, using the nuclear option, re-downloaded and enabled only one mod at any moment.  All other missions/game mechanics work just fine with most mods.  That's at least what I discovered from my 8 hours of testing today.  Sorry buddy
